unit_testing/test_labels.py

```diff
@@ -16,13 +16,22 @@
 
 logger = logging.getLogger(__name__)
 
-seg_img = Image(sct_test_path('t2', 't2_seg-manual.nii.gz'))
-t2_img = Image(sct_test_path('t2', 't2.nii.gz'))
-labels_img = Image(sct_test_path('t2', 'labels.nii.gz'))
+@pytest.fixture(scope="module")
+def seg_img():
+    yield Image(sct_test_path('t2', 't2_seg-manual.nii.gz'))
+
+@pytest.fixture(scope="module")
+def t2_img():
+    yield Image(sct_test_path('t2', 't2.nii.gz'))
+
+@pytest.fixture(scope="module")
+def labels_img():
+    yield Image(sct_test_path('t2', 'labels.nii.gz'))
 
 
 # TODO [AJ] investigate how to parametrize fixtures from test_image.py
 # without redefining the function here
+@pytest.fixture
```

Comment:
For some reason this *couldn't* be `@pytest.fixture(scope="module")`. If I did that, some tests would fail, probably because they were accidentally sharing data. Which is..weird, since the tests all start by, redundantly, making a copy of their inputs, just in case the data is edited in place.

Reply:
there's probably something deeper I'm don't understand about pytest here.